site/templates/film/item.jet

```diff
@@ -92,10 +92,29 @@
             </div>
           {{end}}
           {{if len(film.GetSubtitles()) > 0 }}
-            <div class="meta-detail-subtitle">
-              <h2>{{i18n("meta_detail_subtitles")}}</h2>
-              <p>{{ film.GetSubtitles() }}</p>
-            </div>
+            {{subtitles:= ""}}
+            {{captions:= ""}}
+            {{range film.SubtitleTracks}}
```

Comment:
Based on my reading of [the changelog](https://github.com/shift72/kibble/blob/master/changelog.md#01014):

> Move field Subtitles to field SubtitleTracks. Add new field Subtitles which captures the likes of hard-coded subtitles. Add new function GetSubtitles() for tv episodes, films and bonus content. Populate this with a unique list of SubtitleTracks.Name and Subtitles values.

... you still need to range over `.GetSubtitles()` to ensure you cover both burnt and uploaded subtitles.

Potentially we might want to indicate burnt subs differently...?

Reply:
And yeah, you should check the code is in line with that changelog too.

Reply:
I don't think that hardcoded sub/CCs are returned by the api ? Unless I'm looking in the wrong place 😱 
I've added Czech as HC [to this film]( https://staging-test.shift72.com/admin/#/film/1619/subtitles)
but the film info has no mention of it when using [the api ](https://staging-test.shift72.com/services/meta/v2/film/1619/show_multiple)